.eslintrc.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
+	globals: {
+		appName: true,
```

Comment:
After checking the docs https://eslint.org/docs/user-guide/configuring/language-options#using-configuration-files-1 shouldn't this be set to `'readonly'`?